app/models/server/raw/Roles.js

```diff
@@ -22,4 +22,12 @@ export class RolesRaw extends BaseRaw {
 		}
 		return false;
 	}
+
+	async findUsersInRoles(name, scope, options) {
```

Comment:
you don't need this since you should be calling directly to `Users` model.. see comment at https://github.com/RocketChat/Rocket.Chat/pull/15490/files#diff-a7c02bdeeacb332d7f3eb0b91183e5f6R9